modules/aws/s3.go

```diff
@@ -302,6 +302,40 @@ func GetS3BucketPolicyE(t *testing.T, awsRegion string, bucket string) (string,
 	return aws.StringValue(res.Policy), nil
 }
 
+// GetS3ObjectTagging returns the tags associated to an object stored in a S3 bucket
```

Comment:
```suggestion
// GetS3ObjectTagging returns the tags associated to an object stored in a S3 bucket. This will fail the test if there is an error.
```